deploy/cr.yaml

```diff
@@ -10,6 +10,7 @@ metadata:
 #  - percona.com/delete-backups
 spec:
   crVersion: 2.7.0
+#  initImage: perconalab/percona-postgresql-operator:main
```

Comment:
Let's implement initImage as we did in PSO (user can overwrite globel initImage in each section). 
like:
```
spec:
    backups:
        pgbackrest:
            initImage:
     proxy:
       pgBouncer:
         initImage:
     pmm: 
       initImage:       
```

Reply:
https://github.com/percona/percona-postgresql-operator/pull/1098/commits/018718a775e95405ed19092f7d2396fcf01cd7db